x-pack/solutions/observability/plugins/streams/server/lib/streams/client.ts

```diff
@@ -50,6 +48,9 @@ import {
   deleteStreamObjects,
   deleteUnmanagedStreamObjects,
 } from './stream_crud';
+import { DefinitionNotFoundError } from './errors/definition_not_found_error';
+import { MalformedStreamIdError } from './errors/malformed_stream_id_error';
+import { SecurityError } from './errors/security_error';
```

Comment:
Nit: Any reason to not have an index file in the errors folder?

Reply:
wasn't maintained (only imported like half of the files). (FWIW I'm personally not a big fan of these kinds of imports/exports because of how easy it is to create circular imports and it has a performance impact)

Reply:
> these kinds of imports/exports 

You mean using an index file?

Reply:
yes

Reply:
barrel files is what it's called I think

Reply:
Interesting reading, it's a pity, it makes the code nicer but I can see the cost in tooling, maybe next year we'll have better tree shaking (repeat wishful thinking until the end of time)